examples/chi_sq/src/main.rs

```diff
@@ -11,6 +11,8 @@
 //! Sunscreen's results with other FHE compilers, see
 //! [SoK: Fully Homomorphic Encryption Compilers](https://arxiv.org/abs/2101.07078).
 
+#![allow(clippy::type_complexity)]
```

Comment:
These are scope-able, right? Clippy is complaining that `Cipher<Signed>` is a very complex type definition. I wonder if this is due to Cippy evaluating the expanded type that appears after the `#[fhe_program]` gets expanded, which is `CircuitNode<Cipher<Signed>>`.

If so, we should probably add this allow to `fhe_program_impl` in `sunscreen_comiler_macros/src/fhe_program.rs`. This way, users won't get this clippy warning on our account.